cmd/launcher/watchdog/watchdog_service_windows.go

```diff
@@ -0,0 +1,230 @@
+//go:build windows
+// +build windows
+
+package watchdog
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"os"
+	"time"
+
+	"github.com/kolide/kit/version"
+	agentsqlite "github.com/kolide/launcher/ee/agent/storage/sqlite"
+	"github.com/kolide/launcher/ee/gowrapper"
+	"github.com/kolide/launcher/pkg/launcher"
+	"github.com/kolide/launcher/pkg/log/multislogger"
+	"github.com/kolide/launcher/pkg/log/sqlitelogger"
+	"github.com/pkg/errors"
+	"golang.org/x/sys/windows"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+type winWatchdogSvc struct {
+	systemSlogger, slogger *multislogger.MultiSlogger
+	opts                   *launcher.Options
+}
+
+func RunWatchdogService(systemSlogger *multislogger.MultiSlogger, args []string) error {
+	ctx := context.TODO()
+	systemSlogger.Logger = systemSlogger.Logger.With(
+		"service", launcherWatchdogServiceName,
```

Comment:
Here and below -- I think we usually have a `component` in the log attrs, would we want to standardize here?